lib/services/livesync/livesync-service.ts

```diff
@@ -792,6 +795,10 @@ export class LiveSyncService extends EventEmitter implements IDebugLiveSyncServi
 		}
 	}
 
+	public emitLivesyncEvent (event: string, livesyncData: ILiveSyncEventData): boolean {
+		return this.emit(event, livesyncData);
```

Comment:
Maybe we can add some trace message before emitting the event.

Reply:
Will tracing the event name be enough or should we `JSON.stringify` the livesyncData?

Reply:
You can use:
```TypeScript
this.$logger.trace(`Will emit event ${event} with data`, livesyncData);
```